examples/webpack.config.js

```diff
@@ -7,14 +7,15 @@ const path = require('path');
 const MonacoEditorSrc = path.join(__dirname, '..', 'src');
 
 module.exports = {
-  entry: {
```

Comment:
I believe this creates a separate package for dependencies that don't change often with the `CommonsChunkPlugin`. Why are you commenting out this code?